lib/matplotlib/axes/_base.py

```diff
@@ -2744,8 +2744,6 @@ def ticklabel_format(self, *, axis='both', style='', scilimits=None,
             sb = True
         elif style == 'plain':
             sb = False
-        elif style == 'comma':
-            raise NotImplementedError("comma style remains to be added")
```

Comment:
Can you add an API changes note for this?  I don't think we need to anything about it code-wise, but should document that an exception type has changed.

Reply:
Done.